dfdaemon/downloader/dfget/dfget.go

```diff
@@ -69,22 +69,15 @@ func (dfGetter *DFGetter) getCommand(
 		"-o", output,
 	}
 
-	if dfGetter.config.Notbs {
-		args = append(args, "--notbs")
-	}
-
-	if dfGetter.config.Verbose {
```

Comment:
This should not be delete if the `--verbose` flag of `dfdaemon` has been set.

Reply:
Done. PTAL again. THX